go.mod

```diff
@@ -130,3 +130,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace golang.org/x/net v0.11.0 => golang.org/x/net v0.17.0
```

Comment:
This is not the best way to do that; future updates of the `x/net` version number above (around line 119), even to a version > 0.17.0, would still be replaced by 0.17.0.

IIRC the right way is to `go get golang.org/x/net/@v0.17.0`.